aes/aes-soft/benches/aes128.rs

```diff
@@ -4,6 +4,15 @@ extern crate test;
 use aes_soft::cipher::{BlockCipher, NewBlockCipher};
 use aes_soft::Aes128;
 
+#[bench]
+pub fn aes128_new(bh: &mut test::Bencher) {
+    bh.iter(|| {
+        let cipher = Aes128::new(&Default::default());
+        test::black_box(&cipher);
+    });
+    bh.bytes = 11 * 16;
```

Comment:
I think we can omit `bytes` in this case.

Reply:
Done and rebased.